packages/docs-site/src/components/presenters/sketch-widget/download-icon.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" viewBox="0 0 12 13" xmlns="http://www.w3.org/2000/svg">
+<title>Download Icon</title>
+<desc>Created with Sketch.</desc>
```

Comment:
I'd remove the created with sketch but and the id's that never get used

Reply:
Yeah fair enough - put it through an SVG minifier but for some reason it didn't strip this.